frmts/vrt/vrtprocesseddatasetfunctions.cpp

```diff
@@ -1514,6 +1702,16 @@ void GDALVRTRegisterDefaultProcessedDatasetFuncs()
         GDT_Float64, nullptr, 0, nullptr, 0, LUTInit, LUTFree, LUTProcess,
         nullptr);
 
+    GDALVRTRegisterProcessedDatasetFunc(
+        "BandScaleOffset", nullptr,
+        "<ProcessedDatasetFunctionArgumentsList>"
+        "   <Argument name='scale_{band}' type='builtin' />"
+        "   <Argument name='offset_{band}' type='builtin' />"
+        "</ProcessedDatasetFunctionArgumentsList>",
+        GDT_Float64, BandScaleOffsetSupportedTypes.data(),
```

Comment:
```suggestion
        GDT_Unknown, BandScaleOffsetSupportedTypes.data(),
```

Reply:
code coverage confirms this should be fixed: https://coveralls.io/builds/71575735/source?filename=frmts%2Fvrt%2Fvrtprocesseddatasetfunctions.cpp#L735

Reply:
Fixed, thanks